src/goto-programs/cfg.h

```diff
@@ -31,6 +31,28 @@ struct cfg_base_nodet:public graph_nodet<empty_edget>, public T
   I PC;
 };
 
+/// A multi-procedural control flow graph (CFG) whose nodes store references to
+/// instructions in a goto program.
+///
+/// An instance of cfg_baset<T> is a directed graph whose nodes inherit from a
+/// user-provided type T and store a constant pointer to an instruction of some
```

Comment:
Omit "constant" - there are indeed instantiations of modifiable programs (used by loop acceleration).